superset/assets/src/explore/components/DisplayQueryButton.jsx

```diff
@@ -21,6 +25,7 @@ registerLanguage('json', json);
 const $ = window.$ = require('jquery');
 
 const propTypes = {
+  actions: PropTypes.object.isRequired,
```

Comment:
(optional) I think the code has a fair amount of passing the whole `actions` object around. I think it's preferable to pass something more generic when possible to make the component more reusable and less bound to the app. Here you could pass the `redirectSQLLab` action to an `onOpenInEditor` prop.

Reply:
Good point, will do that.